src/Mod/TechDraw/App/DrawViewClip.cpp

```diff
@@ -73,19 +73,17 @@ void DrawViewClip::addView(App::DocumentObject* docObj)
         return;
     }
 
-    auto* view = dynamic_cast<DrawView*>(docObj);
+    auto* view = static_cast<DrawView*>(docObj);
 
-    if (!view) {
-        auto* link = dynamic_cast<App::Link*>(docObj);
-        if (!link) {
-            return;
-        }
+    auto* link = dynamic_cast<App::Link*>(docObj);
+    if (!link) {
+        return;
+    }
 
-        if (link) {
-            view = dynamic_cast<DrawView*>(link->getLinkedObject());
-            if (!view) {
-                return;
-            }
+    if (link) {

```

Comment:
This `if` is now redundant.

Reply:
The whole App::Link thing might be useless. If DrawViewPart is derived from App::Link, we don't need to cast. If it is not, the cast will always fail. Unreachable code.

Though I suggest fixing this in a separate PR. Author is very new to C++